java/ql/src/experimental/Security/CWE/CWE-326/InsufficientKeySize.qhelp

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>This rule finds uses of encryption algorithms with too small a key size. Encryption algorithms 
+are vulnerable to brute force attack when too small a key size is used.</p>
+    </overview>
+
+    <recommendation>
+        <p>The key should be at least 2048 bits long when using RSA and DSA encryption, 224 bits long when using EC encryption, and 128 bits long when using 
+symmetric encryption.</p>
```

Comment:
Double-check: are these limits still correct? Did the 224 get increased?

Reply:
Correct, it has to be 256.

Reply:
This doesn't look fixed yet.